src/main/java/org/interledger/setup/model/Receiver.java

```diff
@@ -0,0 +1,18 @@
+package org.interledger.setup.model;
+
+import javax.money.CurrencyUnit;
+
+import org.interledger.ilp.InterledgerAddress;
+
+public interface Receiver {
+  
+  InterledgerAddress getAccount();
+  
+  CurrencyUnit getCurrencyUnit();
+
+  int getPrecision();

```

Comment:
If we are strongly typing into CurrencyUnit, is the precision and scale still meaningful? For precision at least, isnt CurrencyUnit.getDefaultFractionDigits equivalent?